packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -56,11 +56,12 @@ const withErrorHandling = (Wrapped) => {
  */
 export const routeComponent = (Wrapped, isPage, locals) => {
     const AppConfig = getAppConfig()
-    const extraArgs = AppConfig.extraGetPropsArgs(locals)
 
     const hocs = AppConfig.getHOCsInUse()
     const getPropsEnabled = hocs.indexOf(withLegacyGetProps) >= 0
 
+    const extraArgs = getPropsEnabled ? AppConfig.extraGetPropsArgs(locals) : {}
```

Comment:
what happens if `getPropsEnabled` is true but `extraGetPropsArgs` is not defined?